pkg/pillar/conntester/conntester.go

```diff
@@ -24,8 +24,33 @@ type RemoteTemporaryFailure struct {
 	WrappedErr error
 }
 
+// PortsNotReady can be returned by TestConnectivity to indicate that one or more
+// ports do not have working connectivity due to a potentially transient error
+// (e.g. missing DNS config, no suitable IP addresses, etc.).
+// For the caller this is a signal to possibly wait and repeat the test later.
+type PortsNotReady struct {
+	WrappedErr error
+	// Ports which are not ready.
+	Ports []string
+}
+
```

Comment:
Wouldn't it make sense to group all of the RemoteTemporyFailure (struct + methods) before all the PortsNotReady struct and methods?

Reply:
makes sense, done